
# HG changeset patch
# User Manu Abraham <abraham.manu@gmail.com>
# Date 1265872265 -14400
# Node ID 3731f71ed6bfc41e76e3dbc9dbde123b10883412
# Parent  ad8b00c9edc2c859268795d8ff486e1de2d2545c
Mantis, hopper: use MODULE_DEVICE_TABLE
use the macro to make modules auto-loadable

Thanks to Ozan ?a?layan <ozan@pardus.org.tr> for pointing it out

From: Manu Abraham <abraham.manu@gmail.com>

Signed-off-by: Manu Abraham <manu@linuxtv.org>

--- linux/drivers/media/dvb/mantis/hopper_cards.c	Mon Mar 08 01:53:08 2010 +0400
+++ linux/drivers/media/dvb/mantis/hopper_cards.c	Thu Feb 11 11:11:05 2010 +0400
@@ -266,6 +266,8 @@
 	{ }
 };
 
+MODULE_DEVICE_TABLE(pci, hopper_pci_table);
+
 static struct pci_driver hopper_pci_driver = {
 	.name		= DRIVER_NAME,
 	.id_table	= hopper_pci_table,
--- linux/drivers/media/dvb/mantis/mantis_cards.c	Mon Mar 08 01:53:08 2010 +0400
+++ linux/drivers/media/dvb/mantis/mantis_cards.c	Thu Feb 11 11:11:05 2010 +0400
@@ -313,6 +313,8 @@
 	{ }
 };
 
+MODULE_DEVICE_TABLE(pci, mantis_pci_table);
+
 static struct pci_driver mantis_pci_driver = {
 	.name		= DRIVER_NAME,
 	.id_table	= mantis_pci_table,

